pennylane/ops/op_math/exp.py

```diff
@@ -171,6 +170,8 @@ def _queue_category(self):
     # pylint: disable=arguments-renamed, invalid-overridden-method
     @property
     def has_decomposition(self):
+        from pennylane.pauli import is_pauli_word  # pylint: disable=import-outside-toplevel
```

Comment:
Why? If there is a circular import I would prefer using `qml.pauli.is_pauli_word` instead

Reply:
I tried it with that syntax first and it still throws the circular import error

Reply:
Are you sure? I just tried it locally and it works

Reply:
Trying again see commit [600663d](https://github.com/PennyLaneAI/pennylane/pull/3391/commits/600663dc07aceea6aa8f73cd4dcc3a530476e8f6)

Reply:
Ahh nice catch it seems to work! I guess it failed for a different reason last time I tried it 🤔